src/org/omegat/filters3/xml/xliff/EditXLIFFOptionsDialog.java

```diff
@@ -95,112 +99,128 @@ public int getReturnStatus() {
-            public void actionPerformed(java.awt.event.ActionEvent evt) {
-                cancelButtonActionPerformed(evt);
-            }
-        });
-        buttonPanel.add(cancelButton);
-
-        getContentPane().add(buttonPanel, java.awt.BorderLayout.SOUTH);
-
-        jPanel1.setBorder(javax.swing.BorderFactory.createEmptyBorder(3, 3, 3, 3));
-        jPanel1.setMinimumSize(new java.awt.Dimension(167, 121));
-        jPanel1.setLayout(new java.awt.GridLayout(0, 1));
-
-        org.openide.awt.Mnemonics.setLocalizedText(jLabel2, OStrings.getString("XLIFF_OPTIONS_LABEL")); // NOI18N
-        jPanel1.add(jLabel2);
-
-        org.openide.awt.Mnemonics.setLocalizedText(compatibility26CB, OStrings.getString("XLIFF_OPTIONS_26")); // NOI18N
-        jPanel1.add(compatibility26CB);
-
-        org.openide.awt.Mnemonics.setLocalizedText(jLabel3, OStrings.getString("XLIFF_OPTIONS_ID")); // NOI18N
-        jPanel1.add(jLabel3);
-
-        buttonGroup1.add(prevAndNextRB);
-        org.openide.awt.Mnemonics.setLocalizedText(prevAndNextRB, OStrings.getString("XLIFF_OPTIONS_ID_NEXT_PREV")); // NOI18N
-        jPanel1.add(prevAndNextRB);
-
-        buttonGroup1.add(transIDRB);
-        org.openide.awt.Mnemonics.setLocalizedText(transIDRB, OStrings.getString("XLIFF_OPTIONS_ID_TRANS_UNIT")); // NOI18N
-        jPanel1.add(transIDRB);
-
-        buttonGroup1.add(transResnameRB);
-        org.openide.awt.Mnemonics.setLocalizedText(transResnameRB, OStrings.getString("XLIFF_OPTIONS_RESNAME_TRANS_UNIT")); // NOI18N
-        jPanel1.add(transResnameRB);
-
-        jPanel2.setLayout(new java.awt.BorderLayout());
-        jPanel2.add(filler1, java.awt.BorderLayout.WEST);
-
-        org.openide.awt.Mnemonics.setLocalizedText(jLabel4, OStrings.getString("XLIFF_OPTIONS_RESNAME_EXPLANATION")); // NOI18N
-        jPanel2.add(jLabel4, java.awt.BorderLayout.CENTER);
-
-        jPanel1.add(jPanel2);
-
-        org.openide.awt.Mnemonics.setLocalizedText(jLabel1, OStrings.getString("XLIFF_OPTIONS_36_ONLY")); // NOI18N
-        jPanel1.add(jLabel1);
-
-        org.openide.awt.Mnemonics.setLocalizedText(forceshortcut2fCB, OStrings.getString("XLIFF_OPTIONS_FORCE2F")); // NOI18N
-        jPanel1.add(forceshortcut2fCB);
-
-        org.openide.awt.Mnemonics.setLocalizedText(ignoreTypeForPhTagsCB, OStrings.getString("XLIFF_OPTIONS_IGNORE4PH")); // NOI18N
-        jPanel1.add(ignoreTypeForPhTagsCB);
-
-        org.openide.awt.Mnemonics.setLocalizedText(ignoreTypeForBptTagsCB, OStrings.getString("XLIFF_OPTIONS_IGNORE4BPT")); // NOI18N
-        jPanel1.add(ignoreTypeForBptTagsCB);
-
-        getContentPane().add(jPanel1, java.awt.BorderLayout.CENTER);
-
-        pack();
-    }// </editor-fold>//GEN-END:initComponents
+        // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
+        private void initComponents() {
+
+                buttonGroup1 = new javax.swing.ButtonGroup();
```

Comment:
Do you have any idea why the indent changed so much here?

Reply:
If it's something to do with tabs vs spaces, could you set your editor to 4 spaces and try to reformat this file?

(You probably would want to use the NetBeans visual editor to make some edit, not do "reformat code" or whatever.)

Reply:
I've used NetBeans 12.0  visual editor to edit the form.
 It may caused by miss configuration (no configuration) of NetBeans.
I'd like to change or flush netbeans configuration then open-save.